hack/update-file-perms.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+find . -name "*.sh" \! -perm /a+x -exec chmod +x {}
```

Comment:
this needs the trailing `+`

Reply:
see: http://man7.org/linux/man-pages/man1/find.1.html

Reply:
```suggestion
find . -name "*.sh" \! -perm /a+x -exec chmod +x {} +
```

Reply:
oops